tests/keras/layers/convolutional_test.py

```diff
@@ -389,6 +389,52 @@ def test_convolution_3d():
 
 
 @keras_test
+def test_conv3d_transpose():
+    filters = 2
+    stack_size = 3
+    num_row = 5
+    num_col = 6
+    num_depth = 7
+
+    for padding in _convolution_paddings:
+        for strides in [(1, 1, 1), (2, 2, 2)]:
+            for kernel_size in [2,3,4]:
```

Comment:
I don't think that's necessary, a single kernel size (3) should be ok. Such a nested loop is very expensive